main.py

```diff
@@ -0,0 +1,36 @@
+from app.books import Book
+from app.displays import ConsoleDisplay, ReverseDisplay
+from app.serializers import JSONSerializer, XMLSerializer
+
+
+def main(book: Book, actions: list) -> None or str:
+
+    actions_dict = {
+        "display": {"console": ConsoleDisplay, "reverse": ReverseDisplay},
+        "print": {
+            "console": (book.title, book.content),
+            "reverse": (book.title, book.content[::-1])
+        },
```

Comment:
I think it's better to make a separate class for book printing as you did with book displaying